01/assets/js/app.js

```diff
@@ -1,4 +1,47 @@
-const a = '4.2';
+const a = "4.2";
 const b = 9;
 
-console.log(a, b);
\ No newline at end of file
+console.log(typeof a);
+console.log(typeof b);
+
+console.log(a, b);
+
+sum = 14.2;
+diff = -5;
+multipl = 52;
+div = 2;
+modulo = 0.6;
+
+if (sum !== parseInt(a) + b) {
+  sum = parseInt(a) + b;
+}
+
+if (diff !== parseInt(a) - b) {
+  diff = parseInt(a) - b;
```

Comment:
Niebyt dobrze to wygląda gdy 2 razy powielasz `parseInt(a)` - możesz wynik tej operacji zapisać do zmiennej i dwukrotnie wykorzystać :)